### DISCUSSION TITLE
float above the Earth an option?

### DISCUSSION THREAD

**Discussion by bblais** [Stellarium/stellarium#1617] [Q&A]
Just wondering if there is a way to set the location to float above the Earth, without the Earth spin, so I can watch the motion of moons and planets without the Earth wobble?  I've searched through the docs and menus and can't find this option.  It has to exist somehow, because there is a solar system observer, but I want to be located where the Earth is.  Maybe I'm missing something easy. 

thanks!

Brian Blais

**Comment by gzotti** [comment]
Not above. Place yourself into the centre of earth by disabling Topocentric Correction.

**Reply by bblais** [reply]
Under Settings/Tools I unchecked "Topcentric coordinates" and I still get the wobble.  Just focus on Jupiter and zoom, and increase speed to see the moon orbits, and you'll see the earth motion.   Mac Stellarium v 0.20.4

**Reply by alex-w** [reply]
> Under Settings/Tools I unchecked "Topcentric coordinates" and I still get the wobble. Just focus on Jupiter and zoom, and increase speed to see the moon orbits, and you'll see the earth motion. Mac Stellarium v 0.20.4

What is “the wobble” for you? And what you expected to see?

**Reply by bblais** [reply]
It's not what I expect to see, really, it's what I'd like to see. I'd like to detach myself from the surface so I can observe from a distance of the Earth but not be influenced by its spinning.  It's really useful if you want to use Stellarium to help teach, say, orbital mechanics -- watch the moons of Jupiter, time them, measure angular displacements, etc...  Years ago when I used Starry Night there was a setting for this, and I've never been able to reproduce the same setting in Stellarium except by moving to the North Pole (latitude 90), but that is a bit of a hack.  I feel it has to be possible, because there is a Solar System Observer which isn't affected by the spin of any planet, but it isn't located at the Earth (which I'd prefer).  

What I see currently is exactly what I'd expect if I was on a spinning planet - a 24 hour back and forth "wobble" of the field of view.  does that help?

**Reply by alex-w** [reply]
> It's not what I expect to see, really, it's what I'd like to see. I'd like to detach myself from the surface so I can observe from a distance of the Earth but not be influenced by its spinning. It's really useful if you want to use Stellarium to help teach, say, orbital mechanics -- watch the moons of Jupiter, time them, measure angular displacements, etc... Years ago when I used Starry Night there was a setting for this, and I've never been able to reproduce the same setting in Stellarium except by moving to the North Pole (latitude 90), but that is a bit of a hack. I feel it has to be possible, because there is a Solar System Observer which isn't affected by the spin of any planet, but it isn't located at the Earth (which I'd prefer).
> 
> What I see currently is exactly what I'd expect if I was on a spinning planet - a 24 hour back and forth "wobble" of the field of view. does that help?

Well, this is may be very strange, but you need to use equatorial mount for it - no need extra hacks or something similar. This is obvious solution. Just toggle mount icon on bottom bar or press Command-M shortcut.

**Reply by gzotti** [reply]
This is not a back and forth wobble (motion of a pendulum), it is a rotation (spin). Simply switch to Equatorial mode then.

To observe Jupiter with its moons spinning fast for several months, you will see the geocentric dance of the planets, also a kind of "wobble". You can use the "Jupiter Observer" as home planet, but its current state is still experimental/not really well aligned to intuitively move the observer position from Jupiter's pole to equator.

**Reply by bblais** [reply]
"switch to Equatorial mode" that did it -- I just didn't associate the vocabulary with what I was trying to do, but it makes total sense.  Thanks!  It was staring at me all this time.
